piker/ui/_display.py

```diff
@@ -1069,6 +1117,12 @@ def update_region_from_pi(
 
             godwidget.resize_all()
 
+            await link_views_with_region(
```

Comment:
this factoring was mostly for sanity and because i don't even really get what layer this whole subsystem fits in 😂